xml/System.Net.Security/SslStream.xml

```diff
@@ -3453,9 +3453,17 @@ Authentication has not occurred.</exception>
         <ReturnType>System.Net.Security.TlsCipherSuite</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the cipher suite resolved to use by this <see cref="T:System.Net.Security.SslStream"/>.</summary>
```

Comment:
```suggestion
        <summary>Gets the cipher suite which was negotiated for this connection.</summary>
```

https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Property says the formula here is `Gets <summary without specifying the type>.`

Reply:
This rule hasn't been enforced in the past but I think it increases readability a lot if we can avoid mentioning types and use more text instead.